website/docs/sysincludedirs.md

```diff
@@ -36,5 +36,6 @@ sysincludedirs { "../includes/**" }
 
 ### See Also ###
```

Comment:
Page should probably just state, alias of `[externalincludedirs](externalincludedirs.md)` and deprecated statut.